zenhub/plugin.spec.yaml

```diff
@@ -3,7 +3,7 @@ extension: plugin
 products: [insightconnect]
 name: zenhub
 title: ZenHub
-description: The ZenHub plugin allows for agile project management in GitHub
+description: Agile project management in GitHub, project road maps, kaban boards all attached to repositories
```

Comment:
```suggestion
description: Agile project management in GitHub, project road maps, and kanban boards all attached to repositories
```

Reply:
still might want to revise this more to make it clear exactly what this plugin is for